cloudflare_exporter/collector.py

```diff
@@ -142,9 +139,12 @@ def collect(self):
 def _get_cloudflare_analytics(token):
     cloudflare = CloudFlare.CloudFlare(debug=False, token=token)
     for zone in cloudflare.zones.get():
-        # get the last 30 minutes (the very last 5 minutes are usually not correct)
         try:
-            series = cloudflare.zones.analytics.colos(zone['id'], params={'since': -35, 'until': -5})
+            # from https://api.cloudflare.com/#zone-analytics-dashboard
+            # By default, range for the Last 60 minutes (from -59 to -1): 1 minute resolution
+            # by default-> the API will move the requested time window backward,
+            #              until it finds a region with completely aggregated data.
+            series = cloudflare.zones.analytics.colos(zone['id'], params={'since': -1})
```

Comment:
Is that really what you want? I'm reading "since 1 minute ago".

Reply:
I update the comment

Reply:
OPT:
Can't see the first patchset, but IMO providing Cloudlfare API link is sufficient.
Eventually you can keep "Retrieve last minute analytics, aggregated using 1-minute resolution"

Reply:
rebase mistake, it's part of the first patchset